### ISSUE TITLE
CLDR: let's talk

### ISSUE THREAD

**Issue by srl295** [phensley/cldr-engine#2]
Hi, @phensley — stumbled on this while trying to get another beta of the cldr json v38 data out.

tl;dr (yes, sounds like cldr) is, let's talk! Can you drop me an email? All my contact info is at <https://github.com/srl295> 

-----
 
Looking at:
- https://github.com/phensley/cldr-engine/commit/bfaf9b691fc3178deb403c19037a4837c2fe8c27
- https://github.com/phensley/cldr-engine/blob/000e108ed662d9ded0a36adc297dab06faace1ac/packages/cldr-compiler/src/cli/downloader/downloader.ts

I'm working on https://unicode-org.atlassian.net/browse/CLDR-14258 right now. 

anyway, i'm working on another drop of the v38 beta CLDR JSON data on NPM. - i'll send an email  to cldr-users when ready…  It's already checked in to the new monorepo on the branch https://github.com/unicode-org/cldr-json/tree/for-38-beta2

**Comment by srl295** [comment]
also, should this engine be listed on http://cldr.unicode.org ? can you open a ticket to do so if so?

**Comment by phensley** [comment]
Thanks @srl295
